src/IB/IBStrategy.cpp

```diff
@@ -91,7 +91,7 @@ namespace IBAMR
 
 /////////////////////////////// PUBLIC ///////////////////////////////////////
 
-IBStrategy::IBStrategy() : d_ib_solver(NULL), d_use_fixed_coupling_ops(false)
+IBStrategy::IBStrategy()
```

Comment:
This doesn't seem related to other C++11 stuff; is it?

Reply:
Clang-tidy moved these initializations to the declaration.